mne/utils.py

```diff
@@ -2015,7 +2016,10 @@ def _get_stim_channel(stim_channel, info, raise_error=True):
     if ch_count > 0:
         return stim_channel
 
-    if 'STI 014' in info['ch_names']:
+    if 'STI101' in info['ch_names']:  # newer Elekta systems
+        return ['STI101']
```

Comment:
In Elekta channel naming, 'STI 014' had a space for some unknown reason, but the new 'STI101' does not.